website/docs/providers/future_provider.mdx

```diff
@@ -36,11 +37,17 @@ Creating the configuration would be done with your typical async/await
 syntax, but inside the provider.
 Using Flutter's asset system, this would be:
 
-<CodeBlock>{trimSnippet(configProvider)}</CodeBlock>
+<AutoSnippet language="dart" {...configProvider}></AutoSnippet>
+
+<ConditionalSnippet codegen={true}>
+  Finally, run the code-generator with{" "}
+  <code>flutter pub run build_runner watch</code>
+</ConditionalSnippet>
+
```

Comment:
I don't think this bit is necessary. It's already mentioned in the getting started doc.

If we mention it in other pages that use codegen too, we'll end-up mentioning it absolutely everywhere